ansible/roles/debops.owncloud/defaults/main.yml

```diff
@@ -815,8 +815,8 @@ owncloud__role_config:
   'updatechecker': '{{ True if (owncloud__variant in ["nextcloud"]) else False }}'
 
   'memcache.local':
-    state: '{{ "present" if (owncloud__apcu_enabled|bool) else "absent" }}'
-    value: '\\OC\\Memcache\\APCu'
+    state: 'present'
+    value: '{{ "\\OC\\Memcache\\Redis" if (owncloud__redis_enabled|bool) else "\\OC\\Memcache\\APCu" }}'
```

Comment:
Sounds good, however you still want to disable it if `owncloud__apcu_enabled` is set to `False`, so something like this:
```yaml
'memcache.local':
  state: '{{ "present" if (owncloud__apcu_enabled|bool or owncloud__redis_enabled|bool) else "absent" }}'
```